### PR TITLE
[magma][feg][cwag][docker] Update docker/deploy to include EAP-SIM service

### PR DESCRIPTION
**Pull request by emakeev**
Signed-off-by: Evgeniy Makeev <evgeniym@fb.com>

## Summary

Update docker/deploy to include EAP-SIM service

## Test Plan

build.py -a
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by mpgermano — COMMENTED** [review]
I think `magmad.yml` and `service_registry.yml` need to be updated for the feg too

File: feg/gateway/deploy/roles/feg_services/files/magma_eap_sim.service [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+SyslogIdentifier=eap_sim
+User=root
+Restart=always
+RestartSec=1s
+StartLimitInterval=0
+MemoryLimit=300M
+
+[Install]
+WantedBy=multi-user.target
+
```
Comment (uri200) [inline feg/gateway/deploy/roles/feg_services/files/magma_eap_sim.service:29]: are you planning to deploy it with dockerless feg?

**Review by uri200 — APPROVED** [review]

File: feg/gateway/deploy/roles/feg_services/files/magma_eap_sim.service (hunk 1 above)
Comment (emakeev) [reply feg/gateway/deploy/roles/feg_services/files/magma_eap_sim.service:29]: not planning to, but I think, SIM should have a parity with AKA

Comment (emakeev) [reply feg/gateway/deploy/roles/feg_services/files/magma_eap_sim.service:29]: not planning to, but added for parity with AKA

**Pull request merged**